model/src/main/scala/hmda/model/institution/InstitutionStatusMessage.scala

```diff
@@ -0,0 +1,6 @@
+package hmda.model.institution
+
+object InstitutionStatusMessage {
```

Comment:
Whenever code depends on a specific string, it's always better to represent it as an object in the system, not as a string. That way the compiler can help us and prevent any mysterious bugs that may come from accidental misspelling or mis-capitalization. In this PR, the main benefit is [here in `InstitutionProtocol.scala`](https://github.com/cfpb/hmda-platform/pull/639/files#diff-3019bb0f3feec594ca0ddf5ebf517947R21), so that when we deserialize JSON, we are certain to match on the correct strings.

Also, this construction parallels the way `SubmissionStatus` uses `SubmissionStatusMessage`.